implementations/rs/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "http_plugin_rs"
```

Comment:
Can we change the name to "polywrap_http_plugin"? This is more inline with python and Javascript package names.